src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -31,6 +32,12 @@ internal sealed partial class DefaultHubDispatcher<THub> : HubDispatcher<THub> w
     private readonly Func<HubLifetimeContext, Exception?, Task>? _onDisconnectedMiddleware;
     private readonly HubLifetimeManager<THub> _hubLifetimeManager;
 
+    [FeatureSwitchDefinition("Microsoft.AspNetCore.SignalR.Hub.CustomAwaitableSupport")]
```

Comment:
My recommendation on feature switches is to name them in such a way in case we ever need to make them as a public API, like we did with JsonSerializer.IsReflectionEnabledByDefault. So that's the pattern I've been following lately.

https://github.com/dotnet/runtime/blob/d9a66070026c9133fade4e61a60d00aec170bfc4/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Helpers.cs#L24-L28.

Reply:
FYI - I'm renaming this switch to `Microsoft.AspNetCore.SignalR.Hub.IsCustomAwaitableSupported` to align with the naming of other [feature switches](https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md):

* System.Diagnostics.Tracing.EventSource.IsSupported
* System.Runtime.CompilerServices.RuntimeFeature.IsDynamicCodeSupported
* System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault
* System.ComponentModel.TypeDescriptor.IsComObjectDescriptorSupported

I just don't want to reset CI right now, so will push that change after CI is done running.